### DISCUSSION TITLE
Hosting on S3 possible?

### DISCUSSION THREAD

**Discussion by ps73** [solidjs/solid-start#395] [Q&A] [answered]
Is it possible to host a solid-start project on s3 without ssr?

When using the auth starter with `ssr: false` I always got 405 errors on some `/_m/routeData...` endpoints, when just opening the index.html from `/dist/public` in the browser.

**Comment by nksaraf** [comment, marked as answer]
That /_m url means that your app might be using some server features. Can you check if your app uses createServerData$. That uses server only features. You should remove that. Then you should be able to deploy to S3 too
